src/System.Linq.Parallel/tests/QueryOperators/AverageTests.cs

```diff
@@ -77,6 +79,16 @@ public static void Average_Long_Longrunning(Labeled<ParallelQuery<int>> labeled,
         }
 
         [Theory]
+        [MemberData("Ranges", (object)(new int[] { 2 }), MemberType = typeof(UnorderedSources))]
+        public static void Average_Long_Overflow(Labeled<ParallelQuery<int>> labeled, int count)
```

Comment:
Can't overflow the addition for Int32, because of the loft to long, you'd run into the iteration limit first, so I didn't initially plan on including it.  I can certainly add it, although it'd be outerloop (take a few minutes to run).

Kicking myself for missing nullable, especially for this change....